src/Sentry.Unity/Integrations/IApplication.cs

```diff
@@ -27,6 +28,8 @@ private ApplicationAdapter()
 
         public string ActiveSceneName => SceneManager.GetActiveScene().name;
 
+        public bool IsSimulator => Application.isEditor;

```

Comment:
Lets map the IApplication 1 to 1 with the Application class from Unity

```suggestion
        public bool IsEditor => Application.isEditor;

```